e2e_test/batch/types/jsonb.slt.part

```diff
@@ -266,3 +266,18 @@ select * from jsonb_each_text('"string"'::jsonb)
 
 statement error cannot deconstruct
 select * from jsonb_each_text('[]'::jsonb)
+
+query TTTTT
+SELECT js,
+  js IS JSON "json?",
+  js IS JSON SCALAR "scalar?",
+  js IS JSON OBJECT "object?",
+  js IS JSON ARRAY "array?"
```

Comment:
What's the suffix "json?" for? I didn't find any explanation in the doc. 👀

Reply:
It is simply column alias, without `AS` keyword but with double quotes so that `?` can be part of the name.

This test is from PostgreSQL doc:
https://www.postgresql.org/docs/16/functions-json.html#FUNCTIONS-SQLJSON-MISC

Reply:
I see. I thought it was a part of the IS JSON expression. 😅